### PR TITLE
Api end points

### PR DESCRIPTION
**Pull request by bvenceslas**

### CODE REVIEW

File: routes/api.js
Comment (guillainbisimwa) [inline routes/api.js:90]: 'Montant` and `somme`?

File: routes/api.js
Comment (guillainbisimwa) [inline routes/api.js:99]: Imaginer dans `register` on pas prevus bcp de details comme 

- [x]  url profile
- [x]  Adresse de livraison par defaut
- [x] sex
On va ajouter ces elements dans l'app mais ne seront pas obligatoires

**Review by guillainbisimwa — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello Josh :wave:

__You've made excellent progress with nice implementation models and API :+1: 
However, some changes are required before it can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

_I hope all the `dates` will be automatically added _

* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Guillain BISIMWA
[Twitter](https://twitter.com/gullain_bisimwa) | [Github](https://github.com/guillainbisimwa) | @tGuillain Bisimwa**
__Happy Coding :smiley:__

File: routes/api.js
Comment (bvenceslas) [reply routes/api.js:99]: done, only verifier url_profile kama njo vile inapashwa kuwa

File: routes/api.js
Comment (bvenceslas) [reply routes/api.js:90]: j'ai enlevé la somme

**Comment by bvenceslas** [comment]
img_documents

**Review by guillainbisimwa — APPROVED** [review]
# Status:  APPROVED  🎉 🚀
#### :clap: :clap:
#### Don't forget to merge and deploy to Heroku! 

gbisimwa

**Pull request merged**
